test/user.test.js

```diff
@@ -28,11 +28,12 @@ describe('User', function() {
   var validMixedCaseEmailCredentials = {email: 'Foo@bar.com', password: 'bar'};
   var invalidCredentials = {email: 'foo1@bar.com', password: 'invalid'};
   var incompleteCredentials = {password: 'bar1'};
+  var validCredentialsUser, validCredentialsEmailVerifiedUser;
 
   // Create a local app variable to prevent clashes with the global
   // variable shared by all tests. While this should not be necessary if
   // the tests were written correctly, it turns out that's not the case :(
-  var app;
+  var app = null;
 
```

Comment:
in fact i meant .eslintrc and the loopback style config: it prevents to have several lines declaring non initialized vars in the same scope
eg. this is not allowed 
```
var x;
var Y;
```
(note: you also have a .jscsrc in loopback though, it notably specifies the max line length)